requirements.txt

```diff
@@ -0,0 +1,17 @@
+git+https://github.com/arcalot/arcaflow-plugin-sdk-python@bce430a59a5782ef26aed9a196149031c667c6ff
+attrs==22.1.0; python_full_version >= "3.7.1" and python_version >= "3.9"
+automat==20.2.0; python_full_version >= "3.7.1" and python_version >= "3.9"
+certifi==2022.9.24; python_version >= "3.6" and python_version < "4"
+constantly==15.1.0; python_full_version >= "3.7.1" and python_version >= "3.9"
+elastic-transport==8.4.0; python_version >= "3.6" and python_version < "4"
+elasticsearch==8.4.3; python_version >= "3.6" and python_version < "4"
```

Comment:
Is this file maintained automatically?  If so, what is the mechanism used to generate this?

If it is maintained manually, what is the process being followed?  This seems fairly labor intensive and prone to error.